webpack/JobWizard/JobWizard.js

```diff
@@ -30,6 +30,7 @@ export const JobWizard = () => {
       effectiveUserValue: responseJob.effective_user?.value || '',
       timeoutToKill: responseJob.job_template?.executionTimeoutInterval || '',
       templateValues: advancedTemplateValues,
+      description: responseJob.job_template.description_format || '',
```

Comment:
broke only the data so eslint wont be angry at the camel case

Reply:
you can add `/* eslint-disable camelcase */` at the top so it won't complain,
and because you need to access the same object several times, you can even do:
```js
useCallback(
    ({ data: { advanced_template_inputs, effective_user, job_template: { executionTimeoutInterval, description_format } } }) => {
```

so it will make the rest a bit more readable